src/commands/wallet/sync.rs

```diff
@@ -587,6 +587,20 @@ fn scan_blocks<P: Parameters + Send + 'static>(
             Ok(true)
         }
         Ok(_) => {
+            if let Some(summary) = db_data.get_wallet_summary(0)? {
+                info!(
```

Comment:
I'd prefer we don't log this if we are in the TUI view, as its log window is limited.